src/services/asset-inventory/cloud-service/cloud-service-detail/modules/CloudServiceLogTab.vue

```diff
@@ -135,27 +138,50 @@ export default defineComponent<Props>({
-                    end: dayjs.utc(props.date),
-                },
-                {
-                    name: 'last2day',
-                    label: i18n.t('INVENTORY.CLOUD_SERVICE.HISTORY.DETAIL.LOG_TAB.LAST_3_DAYS'),
-                    start: dayjs.utc(props.date).subtract(3, 'day'),
-                    end: dayjs.utc(props.date),
-                },
-                {
-                    name: 'last1week',
-                    label: i18n.t('INVENTORY.CLOUD_SERVICE.HISTORY.DETAIL.LOG_TAB.LAST_A_WEEK'),
-                    start: dayjs.utc(props.date).subtract(1, 'week'),
-                    end: dayjs.utc(props.date),
-                },
-            ])),
-            selectedTimeWithin: 'last1day',
+            timeWithinList: computed<PeriodItem[]>(() => {
+                const _date = props.date ? dayjs.utc(props.date) : dayjs.utc();
+                const timeWithinList: PeriodItem[] = [
+                    {
+                        name: 'last1day',
+                        label: i18n.t('INVENTORY.CLOUD_SERVICE.HISTORY.DETAIL.LOG_TAB.LAST_1_DAY'),
+                        start: _date.subtract(1, 'day'),
+                        end: _date,
+                    },
+                    {
+                        name: 'last3days',
+                        label: i18n.t('INVENTORY.CLOUD_SERVICE.HISTORY.DETAIL.LOG_TAB.LAST_3_DAYS'),
+                        start: _date.subtract(3, 'day'),
+                        end: _date,
+                    },
+                    {
+                        name: 'last1week',
+                        label: i18n.t('INVENTORY.CLOUD_SERVICE.HISTORY.DETAIL.LOG_TAB.LAST_A_WEEK'),
+                        start: _date.subtract(1, 'week'),
+                        end: _date,
+                    },
+                    {
+                        name: 'last2weeks',
+                        label: i18n.t('INVENTORY.CLOUD_SERVICE.HISTORY.DETAIL.LOG_TAB.LAST_2_WEEKS'),
+                        start: _date.subtract(2, 'week'),
+                        end: _date,
+                    },
+                    {
+                        name: 'last1month',
+                        label: i18n.t('INVENTORY.CLOUD_SERVICE.HISTORY.DETAIL.LOG_TAB.LAST_1_MONTH'),
+                        start: _date.subtract(1, 'month'),
+                        end: _date,
+                    },
+                    {
+                        name: 'last3months',
+                        label: i18n.t('INVENTORY.CLOUD_SERVICE.HISTORY.DETAIL.LOG_TAB.LAST_3_MONTHS'),
+                        start: _date.subtract(3, 'month'),
+                        end: _date,
+                    },
+                ];
+                if (props.date) return timeWithinList.filter((d) => HISTORY_LOG_PERIOD_LIST.includes(d.name));
+                return timeWithinList.filter((d) => CLOUD_SERVICE_LOG_PERIOD_LIST.includes(d.name));
+            }),
+            selectedTimeWithin: props.date ? 'last1day' : 'last1week',
```

Comment:
suggestion: What about to use constants for consistency?

```suggestion
            selectedTimeWithin: props.date ? HISTORY_LOG_PERIOD_LIST[0] : CLOUD_SERVICE_LOG_PERIOD_LIST[0],
```